src/03_modules.py

```diff
@@ -10,22 +10,25 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print("arguments", sys.argv)
+print("argv", str(sys.argv))
```

Comment:
What it technically was looking for here was a for loop where you printed each item in sys.argv out on its own line.